pkg/test/framework/components/policybackend/policybackend.go

```diff
@@ -84,8 +85,8 @@ spec:
     spec:
       containers:
       - name: app
-        image: {{.Hub}}/test_policybackend:{{.Tag}}
-        imagePullPolicy: {{.ImagePullPolicy}}
+        image: "{{.Hub}}/test_policybackend:{{.Tag}}"
+        imagePullPolicy: IfNotPresent
```

Comment:
Why hard-coding? For the test framework, I would think we would want `Always` by defaultt to ensure we're always using the latest image.